package.json

```diff
@@ -15,10 +15,7 @@
   "eslintConfig": {
     "extends": "@hydrant/eslint-config/modern"
   },
-  "repository": {
-    "type": "git",
-    "url": "git@github.com/hydra-newmedia/hapi-sentry.git"
-  },
+  "repository": "hydra-newmedia/hapi-sentry",
```

Comment:
i know this works but can you change it to `github:hydra-newmedia/hapi-sentry`? Just to be safe

https://docs.npmjs.com/files/package.json#repository